app/static/assets/js/form-validation.js

```diff
@@ -0,0 +1,20 @@
+// Example starter JavaScript for disabling form submissions if there are invalid fields
+(function () {
+  'use strict'
+
+  // Fetch all the forms we want to apply custom Bootstrap validation styles to
+  var forms = document.querySelectorAll('.needs-validation')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/garethtdavies/mina-taxes/pullRequest?prid=7877435)
```suggestion
  var forms = document.querySelectorAll(".needs-validation")
```